code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -62,6 +63,9 @@
 
 	var/obj/item/clothing/accessory/petcollar/collar = null
 	var/can_collar = 0 // can add collar to mob or not
+	var/can_trophy = 0
+	var/trophied = 0
+	var/trophy //It's bad, yeah, but not all simple_animal names correspond to trophy names

```

Comment:
I almost think this should be an object path instead of a string ID, then you could have it create the object when the simple mob is trophied and put it inside the wall mount---then you can just eject it when you remove the trophy from the wall mount--then you're not having to use text2path and also not needlessly creating objects every single time someone removes a trophy from a frame.